### ISSUE TITLE
Leftover reference to "tooltipShow" ?

### ISSUE THREAD

**Issue by sylvinus** [novus/nvd3#2005]
Hello,

In lineChart, it seems there is still a reference to `tooltipShow` and `tooltipHide` which I think are deprecated. Should they be removed?

thanks

**Comment by liquidpele** [comment]
Yes, those can be removed now, it's been at least one release since they were deprecated.
